pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -16,46 +16,47 @@ class BaseBatteryModel(pybamm.BaseModel):
     options: dict
         A dictionary of options to be passed to the model. The options that can
         be set are listed below. Note that not all of the options are compatible with
-        each other and with all of the models implemented in PyBaMM.
+        each other and with all of the models implemented in PyBaMM. Each option is
+        optional and takes a default value if not provided.
 
-            * "cell geometry" : str, optional
+            * "cell geometry" : str
                 Sets the geometry of the cell. Can be "pouch" (default) or
                 "arbitrary". The arbitrary geometry option solves a 1D electrochemical
                 model with prescribed cell volume and cross-sectional area, and
                 (if thermal effects are included) solves a lumped thermal model
                 with prescribed surface area for cooling.
-            * "convection" : bool or str, optional
+            * "convection" : str
                 Whether to include the effects of convection in the model. Can be
-                False (default), "differential" or "algebraic". Must be 'False' for
-                lithium-ion models.
-            * "current collector" : str, optional
+                False (default), "uniform transverse" or "full transverse".
+                Must be "none" for lithium-ion models.
```

Comment:
Is `"none"` the same as `False` (default)? If yes, then they should be called the same.